api/src/pcapi/alembic/versions/20240105T124216_539292753876_alter_venue_columns_to_delete_to_nullable.py

```diff
@@ -0,0 +1,26 @@
+"""Update fieldsUpdated column to be nullable before deletion."""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "539292753876"
+down_revision = "71effb72345c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.alter_column(
+        "venue",
+        "fieldsUpdated",
+        existing_type=postgresql.ARRAY(sa.VARCHAR(length=100)),
+        nullable=True,
+    )
+
+
+def downgrade() -> None:
+    # this column will be set not nullable by fc57ac28e117 post downgrade
```

Comment:
Ce n'est plus le bon numéro non ? 
```suggestion
    # this column will be set not nullable by cbd4d45e39f0 post downgrade
```